.travis.yml

```diff
@@ -1,265 +1,145 @@
 # This will run on Travis' 'new' container-based infrastructure
 sudo: false
 language: c++
+dist: xenial
 
 # Blacklist
 branches:
-    only: 
-        - master
-        - coverity_scan
-    except: gh-pages
+  only: 
+    - master
+    # - coverity_scan
+  except: gh-pages
 
-# Environment variables
 env:
-    global:
-        - GH_REPO_NAME: ComPWA
-        - DOXYFILE: $TRAVIS_BUILD_DIR/doc/Doxyfile.travisCI
```

Comment:
This Doxyfile was deleted. I think that has to be included again. Why does this have to be a global anyway?